### PR TITLE
Added textview to HospitalActivity which displays the amount of devic…

### PR DESCRIPTION
**Pull request by 4ling**
added textview which displays devices overdue for maintenance.

### CODE REVIEW

File: app/src/main/res/layout/activity_hospital.xml [hunk 1]
```diff
@@ -218,6 +218,13 @@
 
     </LinearLayout>
 
+    <TextView
+        android:id="@+id/amountofoverduedevices"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textAlignment="center"
+        android:textSize="15dp"/>
```
Comment (nitelow) [inline app/src/main/res/layout/activity_hospital.xml:226]: Text sizes should be measured in "sp", not "dp"

File: app/src/main/res/layout/activity_hospital.xml [hunk 2]
```diff
@@ -218,6 +218,13 @@
 
     </LinearLayout>
 
+    <TextView
+        android:id="@+id/amountofoverduedevices"
```
Comment (nitelow) [inline app/src/main/res/layout/activity_hospital.xml:222]: please use camelCase or snake_case for naming

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -213,6 +213,7 @@
     <string name="hospital_info_tutorial_state_overview">These symbols give you a quick overview over the states of your devices.</string>
     <string name="hospital_info_tutorial_asset_list">You can browse through a list of your colleagues or devices by expanding the respective section.</string>
     <string name="hospital_info_tutorial_filter">Use this field for filtering.</string>
+    <string name="hospital_info_overdue_devices">devices are scheduled for maintenance</string>
```
Comment (nitelow) [inline app/src/main/res/values/strings.xml:216]: maybe we should use a text that creates more of a sense of urgency?
something like "xy devices should be maintained/serviced immediately"

File: app/src/main/java/ngo/teog/swift/gui/hospital/HospitalActivity.java [hunk 4]
```diff
@@ -7,6 +7,7 @@
 import android.os.Bundle;
 import android.text.Editable;
 import android.text.TextWatcher;
+import android.util.Log;
```
Comment (nitelow) [inline app/src/main/java/ngo/teog/swift/gui/hospital/HospitalActivity.java:10]: please remove unused imports "Log" and "CalendarFragment"

File: app/src/main/java/ngo/teog/swift/gui/hospital/HospitalActivity.java [hunk 5]
```diff
@@ -198,14 +203,45 @@ public void afterTextChanged(Editable editable) {}
...
+                        if (lastMaintenance != null) {
+                            int daysLeft = (device.getMaintenanceInterval()*7)-((int) ((now.getTime() - lastMaintenance.getTime()) / 1000 / 60 / 60 / 24));
+                            if(daysLeft <= 0){
+                                overdueDeviceCount++;
+                            }
+                        }
+                    }
                 }
 
+                overdueDevices.setText(Integer.toString(overdueDeviceCount) + " " + getString(R.string.hospital_info_overdue_devices));
```
Comment (nitelow) [inline app/src/main/java/ngo/teog/swift/gui/hospital/HospitalActivity.java:243]: Conversion Integer.toString(...) is redundant when concatenating an integer with a string using the "+" - operator

File: app/src/main/res/layout/activity_hospital.xml [hunk 6]
```diff
@@ -218,6 +218,13 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
```
Comment (nitelow) [inline app/src/main/res/layout/activity_hospital.xml:224]: View should stand out more against "device_state_overview" view. Maybe add a slightly colored background for the text and a bit of padding?

**Review by nitelow — CHANGES_REQUESTED** [review]
Functionally good, I suggest a few changes in layout and code styling

File: app/src/main/res/layout/activity_hospital.xml (hunk 1 above)
Comment (4ling) [reply app/src/main/res/layout/activity_hospital.xml:226]: done.

File: app/src/main/res/layout/activity_hospital.xml (hunk 2 above)
Comment (4ling) [reply app/src/main/res/layout/activity_hospital.xml:222]: done.

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (4ling) [reply app/src/main/res/values/strings.xml:216]: Current wording was decided on due to user experience. More wording can be implemeted later, when the devices are serviced on a regular basis.

File: app/src/main/java/ngo/teog/swift/gui/hospital/HospitalActivity.java (hunk 4 above)
Comment (4ling) [reply app/src/main/java/ngo/teog/swift/gui/hospital/HospitalActivity.java:10]: done.

File: app/src/main/java/ngo/teog/swift/gui/hospital/HospitalActivity.java (hunk 5 above)
Comment (4ling) [reply app/src/main/java/ngo/teog/swift/gui/hospital/HospitalActivity.java:243]: done

File: app/src/main/res/layout/activity_hospital.xml (hunk 6 above)
Comment (4ling) [reply app/src/main/res/layout/activity_hospital.xml:224]: done

**Review by nitelow — APPROVED** [review]
lgtm

**Pull request merged**